From 8625f895339210a72cbf63d40a915535a497e9b1 Mon Sep 17 00:00:00 2001
From: Markus Niebel <Markus.Niebel@tq-group.com>
Date: Mon, 10 Sep 2018 11:12:07 +0200
Subject: [PATCH 012/188] arm: dt: imx6qdl-mba6: add watchdog out signal

note: we do not enable the signal since the hardware does
not work as-is

Signed-off-by: Markus Niebel <Markus.Niebel@tq-group.com>
---
 arch/arm/boot/dts/imx6qdl-mba6.dtsi | 11 +++++++++++
 1 file changed, 11 insertions(+)

diff --git a/arch/arm/boot/dts/imx6qdl-mba6.dtsi b/arch/arm/boot/dts/imx6qdl-mba6.dtsi
index 1490fb85c1ff..e5487ffe316f 100644
--- a/arch/arm/boot/dts/imx6qdl-mba6.dtsi
+++ b/arch/arm/boot/dts/imx6qdl-mba6.dtsi
@@ -248,6 +248,13 @@
 				MX6QDL_PAD_GPIO_1__USB_OTG_ID  0x00017059
 			>;
 		};
+
+		pinctrl_wdog1: wdog1grp {
+			fsl,pins = <
+				 /* Watchdog out */
+				MX6QDL_PAD_SD1_DAT2__WDOG1_B 0x0000b099
+			>;
+		};
 	};
 };
 
@@ -331,5 +338,9 @@
 };
 
 &wdog1 {
+	pinctrl-names = "default";
+	pinctrl-0 = <&pinctrl_wdog1>;
+	/* does not work on unmodified starter kit */
+	/* fsl,ext-reset-output; */
 	status = "okay";
 };
